Gemfile

```diff
@@ -7,6 +7,7 @@ group :development do
   # gem dependency because we expect to be installed within the
   # Vagrant environment itself using `vagrant plugin`.
   gem "vagrant", :git => "https://github.com/mitchellh/vagrant.git"
+  gem 'iniparse', '~> 1.4', '>= 1.4.2'
```

Comment:
This gem is referred to in `vagrant-aws/config.rb` so I'm not clear on why it is in the `:development` group of this Gemfile instead of as a dependency listed in the gemspec proper.